### PR TITLE
Extensions for amazon-dynamodb, elytron-security-oauth2, smallrye-reactive-type-converters

### PR DESCRIPTION
**Pull request by rsvoboda**
Details for existing extensions:
 * amazon-dynamodb
 * elytron-security-oauth2
 * smallrye-reactive-type-converters

### CODE REVIEW

File: _data/extensions.yaml [hunk 1]
```diff
@@ -238,6 +238,15 @@ categories:
         - nosql
         groupId: io.quarkus
         artifactId: quarkus-neo4j
+      - name: Amazon DynamoDB client
+        description: "A client for the Amazon DynamoDB"
```
Comment (gsmet) [inline _data/extensions.yaml]:
```suggestion
        description: "A client for the Amazon DynamoDB datastore"
```

File: _data/extensions.yaml [hunk 2]
```diff
@@ -410,6 +430,13 @@ categories:
         - security
         groupId: io.quarkus
         artifactId: quarkus-elytron-security
+      - name: Elytron Security OAuth 2.0
+        description: "Secure your services using OAuth 2.0"
+        labels:
+        - security
+        - oauth2
+        groupId: io.quarkus
+        artifactId: quarkus-elytron-security-oauth2
```
Comment (gsmet) [inline _data/extensions.yaml]: Please add this one at the end of the security section. The other ones should be better choices in most of the cases.

File: _data/extensions.yaml [hunk 3]
```diff
@@ -410,6 +430,13 @@ categories:
         - security
         groupId: io.quarkus
         artifactId: quarkus-elytron-security
+      - name: Elytron Security OAuth 2.0
+        description: "Secure your services using OAuth 2.0"
```
Comment (gsmet) [inline _data/extensions.yaml]: This one would be better with `Secure your applications with OAuth2 opaque tokens`

It should be used in very specific cases.

**Comment by rsvoboda** [comment]
@gsmet PR updated

**Comment by gsmet** [comment]
Merged, thanks for taking care of it!

**Pull request merged**